oleesir/authors-haven
# Loggedin user home page

What does this Merge request do?

Implements feat__authenticatedHomePage

Description of Tasks completed?

Implements feat__authenticatedHomePage

How should this be manually tested?

git fetch on your local machine and check out to feat__authenticatedHomePage branch
Run "yarn start" to start the development server
Any background context you want to provide?

N/A

Screenshots (if appropriate)


<img width="1440" alt="Screenshot 2021-12-28 at 7 36 53 PM" src="https://user-images.githubusercontent.com/17679511/147637008-59b0ba4b-4549-4169-b713-989f9dfcf37e.png">
